plugins/ffxiv/ffxiv.cpp

```diff
@@ -0,0 +1,209 @@
+// Copyright 2005-2016 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifdef WIN32
+#include "../mumble_plugin_win32_32bit.h" // Include standard plugin header.
+#else
+#include "../mumble_plugin_linux_32bit.h" // Include standard plugin header.
+#endif
+
+#include "../mumble_plugin_utils.h" // Include plugin header for special functions, like "escape".
+
+#include <cmath>
+
+// Offset values can be obtained from:
+//   http://xivapp.com/api/structures?patchVersion=latest&platform=x86
+// Y is North/South, South is increasing values
+// X is East/West, West is increasing values
+// Heading is in radians, 0 faces south
+
+// Memory offsets
```

Comment:
Ok, i was worried about it after @davidebeatrici mentioned them being mixed.  It's the same in the l4d2 plugin.

Will work on fixing.